playbooks/common/openshift-master/config.yml

```diff
@@ -334,8 +334,8 @@
   serial: 1
   vars:
     sync_tmpdir: "{{ hostvars.localhost.g_master_mktemp.stdout }}"
-    openshift_master_ha: "{{ groups.oo_masters_to_config | length > 1 }}"
-    openshift_master_count: "{{ groups.oo_masters_to_config | length }}"
+    openshift_master_ha: "{{ groups.oo_masters | length > 1 }}"
+    openshift_master_count: "{{ groups.oo_masters | length }}"
```

Comment:
Similar to ^ should we throw an error in openshift_facts if they try to reduce the number of masters? And have a separate playbook for removing masters that would bypass that check?